server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -179,6 +179,11 @@ public static ThreadPoolType fromType(String type) {
 
     public static final double searchAutoscalingEWMA = 0.1;
 
+    // This value is chosen such that a sudden increase in the task durations would need to persist roughly for 60 samples
+    // for the EWMA value to be mostly representative of the increased task durations. Mostly representative means that the
+    // EWMA value is at least within 90% of the new increased task duration.
+    public static final double indexAutoscalingEWMA = 0.04;
```

Comment:
When this came up we were talking about 0.01. I wanted to write this down in terms of number of tasks/samples, and for 0.01 it would take 240 samples to approach the 90%, which seemed pretty high to me. But I'm open to suggestions. The previous value of 0.1 would take 20 samples to reach 90%.

I think one difference between this and the EWMA used for average write load is that this one would need to see N samples (finished tasks), while in average write load we sample every second so we'd have N samples after N seconds.

Reply:
Should we make this configurable? A static setting?

Reply:
We could, but it doesn't seem like something we want to change on specific clusters. But if there is a reason, to do so, I could add a setting.

Reply:
I think there is a value in making it configurable just for the sake of tuning the value (if we ever need to) or rolling back to the prior value if we see it not performing as well as anticipated.

Reply:
I think 0.04 is definitely a safe step forward. I think it could still lead to a single request having high impact, but we can start here.

I'd be fine to merge as is and then possibly make it configurable in a separate PR.